custom_components/omnik_inverter/sensor.py

```diff
@@ -22,6 +34,17 @@
 from . import OmnikInverterDataUpdateCoordinator
 from .const import DOMAIN, MANUFACTURER, SERVICE_DEVICE, SERVICE_INVERTER, SERVICES
 
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Maybe omit it because you don't use it 😄

Reply:
That's bad, there should be some logs here... :)